### ISSUE TITLE
Mismatched icons for harvesting tools

### ISSUE THREAD

**Issue by Archomeda** [arenanet/api-cdi#317]
Some of the harvesting tools icons on the API do not match the actual in-game icons:

### Mining Picks
Item | API icon | Actual icon
---- | -------- | -----------
Copper Mining Pick<br>[(23031)](https://api.guildwars2.com/v2/items/23031) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/b/b0/Orichalcum_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Mining_Pick.png)
Iron Mining Pick<br>[(22995)](https://api.guildwars2.com/v2/items/22995) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/b/b0/Orichalcum_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Mining_Pick.png)
Steel Mining Pick<br>[(23003)](https://api.guildwars2.com/v2/items/23003) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/b/b0/Orichalcum_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Mining_Pick.png)
Darksteel Mining Pick<br>[(23007)](https://api.guildwars2.com/v2/items/23007) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/b/b0/Orichalcum_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Mining_Pick.png)
Mithril Mining Pick<br>[(23010)](https://api.guildwars2.com/v2/items/23010) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/b/b0/Orichalcum_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Mining_Pick.png)
Orichalcum Mining Pick<br>[(23001)](https://api.guildwars2.com/v2/items/23001) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/b/b0/Orichalcum_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Mining_Pick.png)
Master Mining Picks<br>*(apparently retired)*<br>[(23034,23022,23028)](https://api.guildwars2.com/v2/items?ids=23034,23022,23028) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/b/b0/Orichalcum_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Mining_Pick.png)
Pick Axe<br>[(23035)](https://api.guildwars2.com/v2/items/23035) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/b/b0/Orichalcum_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Mining_Pick.png)
Frostbitten Orichalcum Mining Pick<br>[(68376)](https://api.guildwars2.com/v2/items/68376) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/c/c4/Frostbitten_Orichalcum_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Frostbitten_Orichalcum_Mining_Pick.png)
Mining Pick<br>[(23024)](https://api.guildwars2.com/v2/items/23024) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/b/b0/Orichalcum_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Mining_Pick.png)
Advanced Mining Pick<br>[(39114)](https://api.guildwars2.com/v2/items/39114) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/5/50/Advanced_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Advanced_Mining_Pick.png)
Black Lion Mining Pick<br>*(retired)*<br>[(20326)](https://api.guildwars2.com/v2/items/20326) | [![](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png)](https://render.guildwars2.com/file/DB422F44CD5FC897E1255D601736CD4807D2DF22/526358.png) | [![](http://wiki.guildwars2.com/images/d/d0/Black_Lion_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Black_Lion_Mining_Pick.png)
Molten Alliance Mining Pick<br>[(41807,48933)](https://api.guildwars2.com/v2/items?ids=41807,48933) | [![](https://render.guildwars2.com/file/ACF90CA6BCD9CEEFD4D0DCEF2AA87C97B50BDC56/561815.png)](https://render.guildwars2.com/file/ACF90CA6BCD9CEEFD4D0DCEF2AA87C97B50BDC56/561815.png) | [![](http://wiki.guildwars2.com/images/3/3d/Molten_Alliance_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Molten_Alliance_Mining_Pick.png)
Bone Pick<br>[(43527,48932)](https://api.guildwars2.com/v2/items?ids=43527,48932) | [![](https://render.guildwars2.com/file/9827C507181C6B305EA026C2F1EB7448AA390CD5/638367.png)](https://render.guildwars2.com/file/9827C507181C6B305EA026C2F1EB7448AA390CD5/638367.png) | [![](http://wiki.guildwars2.com/images/6/6e/Bone_Pick.png)](http://wiki.guildwars2.com/wiki/File:Bone_Pick.png)
Mine-r-Tron<br>[(67396)](https://api.guildwars2.com/v2/items/67396) | [![](https://render.guildwars2.com/file/64060B1871FBB9472A181BB1F296B9C4D831B728/904685.png)](https://render.guildwars2.com/file/64060B1871FBB9472A181BB1F296B9C4D831B728/904685.png) | [![](http://wiki.guildwars2.com/images/7/74/Mine-r-Tron.png)](http://wiki.guildwars2.com/wiki/File:Mine-r-Tron.png)
Watchwork Mining Pick<br>[(47897)](https://api.guildwars2.com/v2/items/47897) | [![](https://render.guildwars2.com/file/9FE07606DC0F7329DDBE0497293AA7629391040D/711980.png)](https://render.guildwars2.com/file/9FE07606DC0F7329DDBE0497293AA7629391040D/711980.png) | [![](http://wiki.guildwars2.com/images/0/0b/Watchwork_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Watchwork_Mining_Pick.png)
Mad Scientist's Mining Tool<br>[(68905)](https://api.guildwars2.com/v2/items/68905) | [![](https://render.guildwars2.com/file/796209E4012FE25C53D0A04C1DDB0FA5D87079F6/960280.png)](https://render.guildwars2.com/file/796209E4012FE25C53D0A04C1DDB0FA5D87079F6/960280.png) | [![](http://wiki.guildwars2.com/images/3/36/Mad_Scientist%27s_Mining_Tool.png)](http://wiki.guildwars2.com/wiki/File:Mad_Scientist's_Mining_Tool.png)
Firefly Mining Flute<br>[(69958)](https://api.guildwars2.com/v2/items/69958) | [![](https://render.guildwars2.com/file/796209E4012FE25C53D0A04C1DDB0FA5D87079F6/960280.png)](https://render.guildwars2.com/file/796209E4012FE25C53D0A04C1DDB0FA5D87079F6/960280.png) | [![](http://wiki.guildwars2.com/images/2/2c/Firefly_Mining_Flute.png)](http://wiki.guildwars2.com/wiki/File:Firefly_Mining_Flute.png)
Cosmic Mining Tool<br>[(78731)](https://api.guildwars2.com/v2/items/78731) | [![](https://render.guildwars2.com/file/B5BDBEDE1C9ECE63B33E75BA0B1CB8403BCFB401/1431721.png)](https://render.guildwars2.com/file/B5BDBEDE1C9ECE63B33E75BA0B1CB8403BCFB401/1431721.png) | [![](http://wiki.guildwars2.com/images/b/ba/Cosmic_Mining_Tool.png)](http://wiki.guildwars2.com/wiki/File:Cosmic_Mining_Tool.png)

### Logging Axes
Item | API icon | Actual icon
---- | -------- | -----------
Copper Logging Axe<br>[(23030)](https://api.guildwars2.com/v2/items/23030) | [![](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png)](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png) | [![](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)
Iron Logging Axe<br>[(22994)](https://api.guildwars2.com/v2/items/22994) | [![](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png)](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png) | [![](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)
Steel Logging Axe<br>[(23002)](https://api.guildwars2.com/v2/items/23002) | [![](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png)](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png) | [![](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)
Darksteel Logging Axe<br>[(23006)](https://api.guildwars2.com/v2/items/23006) | [![](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png)](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png) | [![](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)
Mithril Logging Axe<br>[(23009)](https://api.guildwars2.com/v2/items/23009) | [![](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png)](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png) | [![](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)
Orichalcum Logging Axe<br>[(23000)](https://api.guildwars2.com/v2/items/23000) | [![](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png)](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png) | [![](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)
Master Logging Axes<br>*(apparently retired)*<br>[(23033,23021,23027)](https://api.guildwars2.com/v2/items?ids=23033,23021,23027) | [![](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png)](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png) | [![](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)
Bun Yan Axe<br>[(23037)](https://api.guildwars2.com/v2/items/23037) | [![](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png)](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png) | [![](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)](http://wiki.guildwars2.com/images/5/54/Orichalcum_Logging_Axe.png)
Splitting Axe<br>[(23023)](https://api.guildwars2.com/v2/items/23023) | [![](https://render.guildwars2.com/file/3CD995B576C8CB39B02D4826EA04690701755ECB/66254.png)](https://render.guildwars2.com/file/3CD995B576C8CB39B02D4826EA04690701755ECB/66254.png) | [![](http://wiki.guildwars2.com/images/b/b0/Orichalcum_Mining_Pick.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Mining_Pick.png)
Frostbitten Orichalcum Logging Axe<br>[(68373)](https://api.guildwars2.com/v2/items/68373) | [![](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png)](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png) | [![](http://wiki.guildwars2.com/images/7/7d/Frostbitten_Orichalcum_Logging_Axe.png)](http://wiki.guildwars2.com/wiki/File:Frostbitten_Orichalcum_Logging_Axe.png)
Advanced Logging Axe<br>[(39112)](https://api.guildwars2.com/v2/items/39112) | [![](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png)](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png) | [![](http://wiki.guildwars2.com/images/d/d7/Advanced_Logging_Axe.png)](http://wiki.guildwars2.com/wiki/File:Advanced_Logging_Axe.png)
Black Lion Logging Axe<br>*(retired)*<br>[(20325)](https://api.guildwars2.com/v2/items/20325) | [![](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png)](https://render.guildwars2.com/file/201CC6E60448CB4BA8AD919C74904EE1F9AEC54F/66597.png) | [![](http://wiki.guildwars2.com/images/e/e3/Black_Lion_Logging_Axe.png)](http://wiki.guildwars2.com/wiki/File:Black_Lion_Logging_Axe.png)
Fused Molten Logging Axe<br>[(67030)](https://api.guildwars2.com/v2/items/67030) | [![](https://render.guildwars2.com/file/15CF97BA7DC573E2BCA852FF1364721B98531470/598608.png)](https://render.guildwars2.com/file/15CF97BA7DC573E2BCA852FF1364721B98531470/598608.png) | [![](http://wiki.guildwars2.com/images/5/56/Fused_Molten_Logging_Axe.png)](http://wiki.guildwars2.com/wiki/File:Fused_Molten_Logging_Axe.png)
Tireless Logging Minion<br>[(67029)](https://api.guildwars2.com/v2/items/67029) | [![](https://render.guildwars2.com/file/20A17F9B7356EE37FEB1003B66D46DBF755B4D91/882250.png)](https://render.guildwars2.com/file/20A17F9B7356EE37FEB1003B66D46DBF755B4D91/882250.png) | [![](http://wiki.guildwars2.com/images/5/54/Tireless_Logging_Minion.png)](http://wiki.guildwars2.com/wiki/File:Tireless_Logging_Minion.png)
Log-r-Tron<br>[(67394)](https://api.guildwars2.com/v2/items/67394) | [![](https://render.guildwars2.com/file/6462A139065B20E25D3626413D080EAC4046B4EE/904683.png)](https://render.guildwars2.com/file/6462A139065B20E25D3626413D080EAC4046B4EE/904683.png) | [![](http://wiki.guildwars2.com/images/2/2c/Log-r-Tron.png)](http://wiki.guildwars2.com/wiki/File:Log-r-Tron.png)
Chop-It-All Logging Axe<br>[(42931,48931)](https://api.guildwars2.com/v2/items?ids=42931,48931) | [![](https://render.guildwars2.com/file/15CF97BA7DC573E2BCA852FF1364721B98531470/598608.png)](https://render.guildwars2.com/file/15CF97BA7DC573E2BCA852FF1364721B98531470/598608.png) | [![](http://wiki.guildwars2.com/images/0/0a/Chop-It-All_Logging_Axe.png)](http://wiki.guildwars2.com/wiki/File:Chop-It-All_Logging_Axe.png)
Mad Scientist's Logging Tool<br>[(68799)](https://api.guildwars2.com/v2/items/68799) | [![](https://render.guildwars2.com/file/AFDEBDF46AE851390BE57EC2CEDAF41703B43C36/983558.png)](https://render.guildwars2.com/file/AFDEBDF46AE851390BE57EC2CEDAF41703B43C36/983558.png) | [![](http://wiki.guildwars2.com/images/8/80/Mad_Scientist%27s_Logging_Tool.png)](http://wiki.guildwars2.com/wiki/File:Mad_Scientist's_Logging_Tool.png)
Swarm Logging Flute<br>[(69921)](https://api.guildwars2.com/v2/items/69921) | [![](https://render.guildwars2.com/file/AFDEBDF46AE851390BE57EC2CEDAF41703B43C36/983558.png)](https://render.guildwars2.com/file/AFDEBDF46AE851390BE57EC2CEDAF41703B43C36/983558.png) | [![](http://wiki.guildwars2.com/images/2/29/Swarm_Logging_Flute.png)](http://wiki.guildwars2.com/wiki/File:Swarm_Logging_Flute.png)
Dreamcleaver Logging Axe<br>[(48955)](https://api.guildwars2.com/v2/items/48955) | [![](https://render.guildwars2.com/file/0334AF66EB5E182C4C0FB2576A31B8A25292E951/665808.png)](https://render.guildwars2.com/file/0334AF66EB5E182C4C0FB2576A31B8A25292E951/665808.png) | [![](http://wiki.guildwars2.com/images/4/4f/Dreamcleaver_Logging_Axe.png)](http://wiki.guildwars2.com/wiki/File:Dreamcleaver_Logging_Axe.png)
Frost Wasp Logging Tool<br>[(48825)](https://api.guildwars2.com/v2/items/48825) | [![](https://render.guildwars2.com/file/0227CB6C96D2AC7FC20B6D465478E9BBF3ED09D9/771066.png)](https://render.guildwars2.com/file/0227CB6C96D2AC7FC20B6D465478E9BBF3ED09D9/771066.png) | [![](http://wiki.guildwars2.com/images/3/3a/Frost_Wasp_Logging_Tool.png)](http://wiki.guildwars2.com/wiki/File:Frost_Wasp_Logging_Tool.png)

### Harvesting Sickles
Item | API icon | Actual icon
---- | -------- | -----------
Copper Harvesting Sickle<br>[(23029)](https://api.guildwars2.com/v2/items/23029) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/9/9b/Orichalcum_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Harvesting_Sickle.png)
Iron Harvesting Sickle<br>[(22992)](https://api.guildwars2.com/v2/items/22992) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/9/9b/Orichalcum_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Harvesting_Sickle.png)
Steel Harvesting Sickle<br>[(23004)](https://api.guildwars2.com/v2/items/23004) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/9/9b/Orichalcum_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Harvesting_Sickle.png)
Darksteel Harvesting Sickle<br>[(23005)](https://api.guildwars2.com/v2/items/23005) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/9/9b/Orichalcum_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Harvesting_Sickle.png)
Mithril Harvesting Sickle<br>[(23008)](https://api.guildwars2.com/v2/items/23008) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/9/9b/Orichalcum_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Harvesting_Sickle.png)
Orichalcum Harvesting Sickle<br>[(22997)](https://api.guildwars2.com/v2/items/22997) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/9/9b/Orichalcum_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Harvesting_Sickle.png)
Master Harvesting Sickles<br>*(apparently retired)*<br>[(23032,23020,23026)](https://api.guildwars2.com/v2/items?ids=23032,23020,23026) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/9/9b/Orichalcum_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Harvesting_Sickle.png)
Worker's Sickle<br>[(23036)](https://api.guildwars2.com/v2/items/23036) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/9/9b/Orichalcum_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Harvesting_Sickle.png)
Frostbitten Orichalcum Harvesting Sickle<br>[(68377)](https://api.guildwars2.com/v2/items/68377) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/7/7e/Frostbitten_Orichalcum_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Frostbitten_Orichalcum_Harvesting_Sickle.png)
Foraging Sickle<br>[(23025)](https://api.guildwars2.com/v2/items/23025) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/9/9b/Orichalcum_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Orichalcum_Harvesting_Sickle.png)
Advanced Harvesting Sickle<br>[(39108)](https://api.guildwars2.com/v2/items/39108) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/4/4d/Advanced_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Advanced_Harvesting_Sickle.png)
Black Lion Harvesting Sickle<br>*(retired)*<br>[(20324)](https://api.guildwars2.com/v2/items/20324) | [![](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png)](https://render.guildwars2.com/file/EFBB674F9965A2304CBAC39376230396D73EBCFD/534199.png) | [![](http://wiki.guildwars2.com/images/1/13/Black_Lion_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Black_Lion_Harvesting_Sickle.png)
Fused Molten Sickle<br>[(67032)](https://api.guildwars2.com/v2/items/67032) | [![](https://render.guildwars2.com/file/51782971EFC6B80F4B5BD0FB080D0CFFA37B1576/866830.png)](https://render.guildwars2.com/file/51782971EFC6B80F4B5BD0FB080D0CFFA37B1576/866830.png) | [![](http://wiki.guildwars2.com/images/1/1b/Fused_Molten_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Fused_Molten_Sickle.png)
Tireless Harvesting Minion<br>[(67063)](https://api.guildwars2.com/v2/items/67063) | [![](https://render.guildwars2.com/file/66A0544916B86F1E00F173ABB9555FC6FDCECBE0/882264.png)](https://render.guildwars2.com/file/66A0544916B86F1E00F173ABB9555FC6FDCECBE0/882264.png) | [![](http://wiki.guildwars2.com/images/4/4b/Tireless_Harvesting_Minion.png)](http://wiki.guildwars2.com/wiki/File:Tireless_Harvesting_Minion.png)
Reap-r-Tron<br>[(67395)](https://api.guildwars2.com/v2/items/67395) | [![](https://render.guildwars2.com/file/AB92677C955A7ED5A5D25662C6F21C68F26245A1/904684.png)](https://render.guildwars2.com/file/AB92677C955A7ED5A5D25662C6F21C68F26245A1/904684.png) | [![](http://wiki.guildwars2.com/images/c/cf/Reap-r-Tron.png)](http://wiki.guildwars2.com/wiki/File:Reap-r-Tron.png)
Thresher-Sickle 5000<br>[(49308)](https://api.guildwars2.com/v2/items/49308) | [![](https://render.guildwars2.com/file/17EED9D8F570EF1EF00045533F65CD791EA4D0FB/582347.png)](https://render.guildwars2.com/file/17EED9D8F570EF1EF00045533F65CD791EA4D0FB/582347.png) | [![](http://wiki.guildwars2.com/images/9/98/Thresher-Sickle_5000.png)](http://wiki.guildwars2.com/wiki/File:Thresher-Sickle_5000.png)
Mad Scientist's Harvesting Tool<br>[(69478)](https://api.guildwars2.com/v2/items/69478) | [![](https://render.guildwars2.com/file/BBC7E4B707C3624DEFB22C62FC2C9390741F090D/978010.png)](https://render.guildwars2.com/file/BBC7E4B707C3624DEFB22C62FC2C9390741F090D/978010.png) | [![](http://wiki.guildwars2.com/images/d/d8/Mad_Scientist%27s_Harvesting_Tool.png)](http://wiki.guildwars2.com/wiki/File:Mad_Scientist's_Harvesting_Tool.png)
Butterfly Harvesting Flute<br>[(70044)](https://api.guildwars2.com/v2/items/70044) | [![](https://render.guildwars2.com/file/BBC7E4B707C3624DEFB22C62FC2C9390741F090D/978010.png)](https://render.guildwars2.com/file/BBC7E4B707C3624DEFB22C62FC2C9390741F090D/978010.png) | [![](http://wiki.guildwars2.com/images/d/d1/Butterfly_Harvesting_Flute.png)](http://wiki.guildwars2.com/wiki/File:Butterfly_Harvesting_Flute.png)
Consortium Harvesting Sickle<br>[(42594,48930)](https://api.guildwars2.com/v2/items?ids=42594,48930) | [![](https://render.guildwars2.com/file/B8FAB3CF449D4BF50BD29AEA7B1A753F7E5CE792/643281.png)](https://render.guildwars2.com/file/B8FAB3CF449D4BF50BD29AEA7B1A753F7E5CE792/643281.png) | [![](http://wiki.guildwars2.com/images/3/36/Consortium_Harvesting_Sickle.png)](http://wiki.guildwars2.com/wiki/File:Consortium_Harvesting_Sickle.png)
Jack-in-the-Box Scythe<br>[(44876,48934)](https://api.guildwars2.com/v2/items?ids=44876,48934) | [![](https://render.guildwars2.com/file/B8FAB3CF449D4BF50BD29AEA7B1A753F7E5CE792/643281.png)](https://render.guildwars2.com/file/B8FAB3CF449D4BF50BD29AEA7B1A753F7E5CE792/643281.png) | [![](http://wiki.guildwars2.com/images/e/e9/Jack-in-the-Box_Scythe.png)](http://wiki.guildwars2.com/wiki/File:Jack-in-the-Box_Scythe.png)

**Comment by lye** [comment]
I think the main issue is that the icon provided by the item, in the case of gathering tools, is almost always overridden by the `default_skin`'s icon. IIRC there was something super weird with the gathering tool skins that caused stuff to blow up -- will have to figure out what that was and see if I fixed it at some point.

Then I need to go through and make sure all `default_skin`s referenced by `/v2/items` are properly in the `/v2/skins` endpoint. That should fix this up.

**Comment by lye** [comment]
Okay, hmm, the default skins are *also* the wrong icon. WTF.

**Comment by lye** [comment]
Okay, found the issue. At some point I added code to override the icon to be the default skin's icon (if a default skin existed) because it was confusing to have wrong icons. Unfortunately, it *seems* that gathering tools don't use the same system -- their default skins *don't* override their icon. This is true for all items whose type isn't `Back`, `Armor` or `Weapon`, so I've adjusted the code locally to strip off the `default_skin` field for all but those types to mimic what the client UI does.

I think that should resolve the issue.
